app/services/qa/linked_data/authority_url_service.rb

```diff
@@ -37,29 +42,36 @@ def action_request_variable(action_config, action)
             action_config.qa_replacement_patterns[key]
           end
 
-          def supports_subauthorities?(action_config)
```

Comment:
Since this is only called once from a private method, it is collapsed into the caller.